frontend_tests/node_tests/settings_user_groups.js

```diff
@@ -154,11 +154,11 @@ run_test('populate_user_groups', () => {
         assert(config.fixed);
         assert(config.dropup);
         assert(config.stopAdvance);
-        assert.equal(typeof(config.source), 'function');
-        assert.equal(typeof(config.highlighter), 'function');
-        assert.equal(typeof(config.matcher), 'function');
-        assert.equal(typeof(config.sorter), 'function');
-        assert.equal(typeof(config.updater), 'function');
+        assert.equal(typeof (config.source), 'function');
```

Comment:
At first, these `typeof` fixes seem bit weird to me but I it becomes clear as `typeof` isn't a function it's an operator :sweat_smile:

Reply:
Should we get rid of the parens, then?

Reply:
Okay I'll do it then!

Reply:
Cool.  I'm going to go ahead and merge this, since I've finished reviewing it; that can be a follow-up commit.